README.md

```diff
@@ -1,4 +1,4 @@
-# VM UI
+# Player.Vm.Ui
```

Comment:
If you're updating these (and feel like they're done) you can also resolve those README tickets

Reply:
The READMEs still need a lot of work I would say.